external/common/include/fast_cpp_csv_parser/README.md

```diff
@@ -0,0 +1,268 @@
+# Fast C++ CSV Parser
```

Comment:
Also, how should the include subdir be called? `external/common/include/fast_cpp_csv_parser/` I kept the name of the original project, this is also used in `#include`